cli/src/pcluster/config/cfn_param_types.py

```diff
@@ -883,6 +884,19 @@ def refresh(self):
             self.value = get_default_instance_type()
 
 
+class IamLambdaRoleCfnParam(CfnParam):
+    """
+    Class to manage the iam_lambda_role parameter.
+
+    We need this class in order to convert role name to arn.
+    """
+
+    def refresh(self):
+        """Covert the value to arn if the value is not arn."""
+        if self.value and not self.value.startswith("arn"):
+            self.value = boto3.client("iam").get_role(RoleName=self.value)["Role"]["Arn"]
```

Comment:
some users might not be granted a policy to perform a GetRole. can we compute the arn without doing this call? 
(let's pay attention to the partition here aws/aws-cn/aws-gov)